docs/docs/rad-lab-ui/troubleshooting.md

```diff
@@ -4,3 +4,20 @@ sidebar_position: 4
 # Troubleshooting
 
 Please feel free to reach out and to create a [GitHub issue](https://github.com/GoogleCloudPlatform/rad-lab/issues) in case of any issues or concerns.
+
+## Email Notifications
```

Comment:
I think the best place for this section is under  [Global Admin Variables](https://googlecloudplatform.github.io/rad-lab/docs/rad-lab-ui/ui_installation/publishing-modules/#global-admin-variables) section compared to Troubleshooting section.

And also we should remove the mention of `Preferred Google Cloud Region and Zone` from Global Admin Variables.

Reply:
Moved `Email Notifications` section to `Global Admin Variables`